lib/executor.js

```diff
@@ -87,6 +88,10 @@ function assertLoopBackVersion(app) {
   }
 }
 
+function setEnv(app, instructions) {
+  app.set('env', instructions.env || 'development');
+}
+
```

Comment:
@bajtos: would you have some insights to why the value of `instructions.env` becomes undefined in test cases in `executor.test.js` script , so that's why I set the default value to 'development' here:

Also it is worth mentioning in this [test script](http://ci.strongloop.com/job/loopback-boot/577/label=ubuntu-0.12/testReport/junit/executor%20with%20middleware/json/should_parse_multiple_config_variables/) it expects `test` as opposed to `development` for the [this test line](https://github.com/strongloop/loopback-boot/blob/master/test/executor.test.js#L456)